README.md

```diff
@@ -177,6 +192,40 @@ decoder.decode("barilla.js", 0, 4);  // orzo
 decoder.decode("barilla.js", 0, 25); // penne
 ```
 
+## Contributions
+
+We :heart: contributions.
+
+Have you had a good experience with pasta-sourcemaps? Why not share some love
+and contribute code, or just let us know about any issues you had with it?
+
+We welcome issue reports
+[here](https://github.com/bloomberg/pasta-sourcemaps/issues); be sure to choose
```

Comment:
Opted to use absolute paths to Issues/Pull Requests, relative paths don't appear to work from some locations.